src/typer/dokumentApi.ts

```diff
@@ -90,3 +90,837 @@ export interface IVerdiliste {
   verdiliste?: IVerdiliste[];
   alternativer?: string;
 }
+
+export interface IDokumentData {
+  behandling: IBehandling;
+  vilkår: IVilkår;
+  personopplysninger: IPersonopplysninger;
+  vedtak: IVedtak;
+  søknadsdatoer?: ISøknadsdatoer;
+}
```

Comment:
Burde ikke hele denne bolken legges i en egen fil? `blankettApi` for eksempel?

Jeg ser at nå exporterer denne filen to interfaces med samme navn for eksempel.. (IDokumentData)

Reply:
fiksa denne selv [274cdc5](https://github.com/navikt/familie-brev/pull/537/commits/274cdc5b9bbc7110d3e40abe1bb43a102ca35133)